### PR TITLE
fix: fixed guest fields

### PR DESCRIPTION
**Pull request by duypv98**

### CODE REVIEW

File: core/guests.core.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ async function findGuestById(guestId) {
  */
 async function saveNewGuestWithEventId(userId, eventId, guestInfo) {
   const {
-    email, phoneNumber, gender, major, something,
+    email, fullName, phoneNumber, gender, major, answer,
```
Comment (namdaoduy) [inline core/guests.core.js]: Remove major please

Comment (namdaoduy) [reply core/guests.core.js]: `answers` not `answer`

**Review by namdaoduy — CHANGES_REQUESTED** [review]

File: core/guests.core.js (hunk 1 above)
Comment (duypv98) [reply core/guests.core.js]: solved https://github.com/EasyEventV2/ee2_api/pull/12/commits/05d70661939a65148fa8a6024756c7fe05d6ba49

Comment (duypv98) [reply core/guests.core.js]: > Remove major please

https://github.com/EasyEventV2/ee2_api/pull/12/commits/87a84ff45ece9b7540cb6d699f210ed1aa795eb8

**Review by namdaoduy — APPROVED** [review]

**Pull request merged**